.drone.yml

```diff
@@ -1492,6 +1493,8 @@ steps:
   - C:\App\grabpl.exe init-enterprise C:\App\grafana-enterprise
```

Comment:
I think we might need to pass $DRONE_TAG here, similarly to how it's done in 1102? Otherwise, we'll actually build with the `main` branch, afaict 🤔

Reply:
Makes sense, I'll take a look after fixing the docker images.